api/api/src/main/scala/uk/ac/wellcome/platform/api/QueryParams.scala

```diff
@@ -29,9 +30,13 @@ object SingleWorkParams extends QueryParamsUtils {
     parameter(
       (
         "include".as[V2WorksIncludes].?,
+        "expandPaths".as[List[String]].?,
```

Comment:
Not sure this parameter name makes very much sense to me?

Why not just have it in the includes?

Reply:
It's different to includes:

* `includes=collection` means we display the tree, with the current work expanded in the tree

* `expandPaths=a/b/c,a/x/y` means we also want to expand `a/b/c` and `a/x/y` in the returned tree